### PR TITLE
Actually remove add_sequence method

### PR DESCRIPTION
**Pull request by peterjc**
- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

See discussion https://github.com/biopython/biopython/commit/422b5051624bcf3c0c145e9e951f551d86b52931#r50771274

Pull request #922 intended to remove the method, but accidentally restored it as part of removing
the legacy base class. The add_sequence zombie method has since stayed in place since Biopython 1.69

### CODE REVIEW

**Comment by peterjc** [comment]
Should have checked, we do still have some tests using ``add_sequence``

File: Bio/Align/__init__.py [hunk 1]
```diff
@@ -1620,8 +1570,8 @@ def map(self, alignment):
         >>> alignment1 = alignments1[0]
         >>> print(alignment1)
         AAAAAAAACCCCCCCAAAAAAAAAAAGGGGGGAAAAAAAA
-                |||||||-----------||||||        
-                CCCCCCC-----------GGGGGG
```
Comment (peterjc) [inline Bio/Align/__init__.py:1624]: My editor was "helpful", but this kind of trailing whitespace is really annoying in doctests.

**Comment by peterjc** [comment]
Having realised the "zombie" method was still in use in the docstrings, the chances are a bit higher that people are still using it. Maybe we should do a deprecation cycle... ping @MarkusPiotrowski for another set of eyes?

**Review by mdehoon — APPROVED** [review]

**Comment by mdehoon** [comment]
> Having realised the "zombie" method was still in use in the docstrings, the chances are a bit higher that people are still using it.

If you are concerned about this, you could ask on the mailing list whether anybody is using this method.

**Comment by peterjc** [comment]
Done, https://mailman.open-bio.org/pipermail/biopython/2021-May/016870.html

I'm happy to rework this PR into deprecating the method, and removing it from the other docstring examples.

**Comment by mdehoon** [comment]
No replies on the mailing list - time to press the button.

**Comment by peterjc** [comment]
OK then - will you do the honours?

**Pull request merged**